cl/citations/parenthetical_utils.py

```diff
@@ -47,3 +48,43 @@ def create_parenthetical_groups(cluster: OpinionCluster) -> None:
         )
         group_to_create.save()
         group_to_create.parentheticals.set(cg.parentheticals)
+
+
+def disconnect_parenthetical_group_signals():
+    """Disconnect ParentheticalGroup ES indexing on save and delete
```

Comment:
This seems OK, but it should have a loud warning that it disconnects the signals across the entire application or else somebody will think it's safe to use when it's not.